### PR TITLE
Add IST0161 message description

### PR DESCRIPTION
**Pull request by hanxiaop**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [x] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by hanxiaop** [comment]
@ericvn PTAL, thanks

File: content/en/docs/reference/config/analysis/ist0161/index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+If the secret is missing, create a new secret with the correct TLS certificate and private key. For example, using `kubectl`:
+
+{{< text bash >}}
+$ kubectl create secret tls my-tls-secret --cert=path/to/cert.pem --key=path/to/key.pem -n <namespace>
+{{< /text >}}
+
+If the secret is found but the TLS certificate is invalid, update the secret with the correct TLS certificate and private key.
+
+Make sure to replace `<namespace>` with the actual namespace where the Gateway workload is running, and update the file paths to point to the correct certificate and key files.
```
Comment (ericvn) [inline content/en/docs/reference/config/analysis/ist0161/index.md:28]: Does it make sense to switch these last two sentences around? The last sentence references the `text bash` while the prior sentence talks about the issue where the issue was found.
```suggestion
Make sure to replace `<namespace>` with the actual namespace where the Gateway workload is running, and update the file paths to point to the correct certificate and key files.

If the secret is found but the TLS certificate is invalid, update the secret with the correct TLS certificate and private key.
```

**Comment by ericvn** [comment]
Any comments from @istio/wg-user-experience-maintainers?

File: content/en/docs/reference/config/analysis/ist0161/index.md (hunk 1 above)
Comment (hanxiaop) [reply content/en/docs/reference/config/analysis/ist0161/index.md:28]: Yes it makes sense, thanks for the suggestion!

**Comment by hanxiaop** [comment]
> Any comments from @istio/wg-user-experience-maintainers?

The original PR is https://github.com/istio/istio/pull/43921

**Review by ericvn — APPROVED** [review]
Thanks.

**Pull request merged**